src/pages/my_page/user_profile/UserIdentity.tsx

```diff
@@ -50,7 +49,6 @@ export const UserIdentity = ({ user, hasActiveEmployment }: UserIdentitiesProps)
             <NationalIdNumberField nationalId={nationalId} />
             <TextField
               data-testid={dataTestId.myPage.myProfile.cristinIdField}
-              size="small"
               label={'Cristin ID'}
```

Comment:
Ups, denne skulle vært gjennom oversettelse! (selv om den blir lik for norsk og engelsk)